source/instance.txt

```diff
@@ -11,46 +11,37 @@ Compass Home
    :class: singlecol
 
 The Compass Home screen provides details regarding the
-MongoDB instance to which Compass is connected, including:
+MongoDB instances to which Compass is connected. 
 
-- The connection name if the connection is a favorite connection,
-  or "My Cluster" otherwise.
+.. note::
 
-- Your :ref:`saved queries and aggregation pipelines <view-favorite-queries>`. 
-
-- A list of the instance's :doc:`databases </databases>`.
+   Starting in version 1.44.0, you can connect to multiple MongoDB
+   instances at once through Compass. When you activate multiple
+   connections, Compass displays each active connection in a separate
+   tab. 
 
-- :doc:`Performance statistics </performance>`, including memory
-  usage, operation counts, and slowest operations.
-
-Once connected to a MongoDB instance, you can reach the Compass
-Home screen by clicking the cluster name in the upper left
-corner.
+The Compass Home screen includes the following sections:
 
-Databases Tab
--------------
+- :ref:`Connections Sidebar <connections-sidebar>`
 
-The :guilabel:`Databases` tab of the Home screen shows a list of all
-the databases on the current connection, with each one's storage size,
-number of collections, and number of indexes. For more information
-about the :guilabel:`Databases` tab and instructions on viewing
-a database, see :doc:`Databases </databases>`.
-
-.. _performance-tab-home:
+- Your :ref:`saved queries and aggregation pipelines <view-favorite-queries>`. 
 
+Once connected to a MongoDB instance, you can reach the Compass
+Home screen by clicking a cluster name in the :guilabel:`Connections
+Sidebar`. 
 
-Performance Tab
----------------
+Connections Sidebar
+-------------------
 
-The :guilabel:`Performance` tab provides real-time server
-performance statistics and graphs, showing insights such as:
+The top section of the sidebar shows the active MongoDB instances to which
+Compass is connected. 
 
-- Which database collections receive the most operations
-- Which operations take the longest to execute
-- Memory utilization
+The bottom section of the sidebar lists your saved and favorite
+connections. The menu shows both active and inactive connections. To
+activate a connection, click the name of the connection. 
 
-For more information about the :guilabel:`Performance` tab, see
```

Comment:
I couldn't find the file here, but reviewing this section made me realize that we need to make a small tweak on the "Performance" page, since we changed the entry point as part of this project. Could you make that tweak before closing the project? It's really just the top line of this page and the screenshots: https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-38430/performance/

Reply:
I'll add this to [DOCSP-42696](https://jira.mongodb.org/browse/DOCSP-42696)!